### ISSUE TITLE
WebsiteAgent support fetch html page after executing javascript?

### ISSUE THREAD

**Issue by waldens** [cantino/huginn#937]
Like this WebsiteAgent:
```
{
  "expected_update_period_in_days": "14",
  "url": "http://news.sina.com.cn/xz/",
  "type": "html",
  "mode": "on_change",
  "extract": {
    "url": {
      "css": ".feed-card-item>h2>a",
      "value": "@href"
    }
  }
}
```
Only after enabling JS, main body of this page are shown and the `.feed-card-item>h2>a` can be scrapped. but does WebsiteAgent can excute JS and then scratch this `.feed-card-item>h2>a` part? If not, is there any way else to get this part of the page? like JavaScriptAgent or else? Thanks for help!

**Comment by cantino** [comment]
Hi @waldens.  At the moment the WebsiteAgent does not support running any JavaScript on the page.  There may be an external web service that could do this, or you could look at integrating phantomjs (or similar) into Huginn.

**Comment by irfancharania** [comment]
Duplicate of issue #888

**Comment by irfancharania** [comment]
Also, for that site, I wonder if you might be able to generate what you need directly from the json used to generate that feed?
http://feed.mix.sina.com.cn/api/roll/get?pageid=9&lid=141&num=15&page=1

**Comment by waldens** [comment]
Wow Thanks @irfancharania! It's worked and soooo helped!

**Comment by cantino** [comment]
:smile:
